tests/unit_tests/config/test_forward_model.py

```diff
@@ -305,3 +307,300 @@ def test_that_eclipse100_require_version_field():
         ConfigValidationError, match="Required keyword <VERSION>.*ECLIPSE100"
     ):
         _ = ErtConfig.from_file(test_config_file_name)
+
+
+def test_that_plugin_forward_models_are_installed():
+    test_config_contents = dedent(
+        """
+        NUM_REALIZATIONS  1
+        FORWARD_MODEL PluginForwardModel(<arg1>=hello,<arg2>=world,<arg3>=derpyderp)
+        """
+    )
+    with open("test.ert", "w", encoding="utf-8") as fh:
```

Comment:
This will write the config in the root directory of the project. You need to use either the `use_tmpdir` fixture or `tmpdir` or `tmp_path`. I suggest the following:

```python
def test_that_plugin_forward_models_are_installed(tmp_path):
   (tmp_path / "config.ert").write_text(dedent(
        """
        NUM_REALIZATIONS  1
        FORWARD_MODEL PluginForwardModel(<arg1>=hello,<arg2>=world,<arg3>=derpyderp)
        """
        )
    )
```

Reply:
ooopsie! fixing